002-secrets.yaml

```diff
@@ -11,7 +11,7 @@ data:
   GODADDY_API_KEY: ZExxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxUOXo=
   GODADDY_API_SECRET: WExxxxxxxxxxxxxxxxxxxxxxxxxxxxx==
 ---
-# https://docs.traefik.io/middlewares/basicauth/#users
+# https://doc.traefik.io/traefik/middlewares/http/basicauth/#usersfile
```

Comment:
Can you please change this to “#users” instead of “#usersfile”, the comment doesn’t provide any info on creating a secret from a file.

Reply:
Done! Good catch @sleighzy